src/imo/IMBOR2022_Attributen.rq

```diff
@@ -0,0 +1,34 @@
+# Alle attributen met hun directe informatie
+
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX coll: <https://data.crow.nl/rest-api/id#>
+PREFIX restapi: <https://data.crow.nl/rest-api/def#>
+PREFIX imbor: <https://data.crow.nl/imbor/def/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX nen2660: <https://w3id.org/nen2660/def#>
+
+SELECT DISTINCT ?uriConcept ?labelConcept ?definitieConcept ?toelichtingConcept ?uriSupertype ?uriGrootheid
+
+WHERE { 
+    
+# Selecteer alle attributen en optioneel hun parents.   
+	?uriConcept a rdf:Property .
+    OPTIONAL {?uriConcept rdfs:subPropertyOf ?uriSupertype }.
+    
+# Kies alleen degene die voor de REST-API gedefinieerd zijn als "IMBOR", "versie 2022" en in de "IMBOR Kern" zitten.
+# (Omdat het hostingplatform van het SPARQL-endpoint geen grafen onderscheidt, is deze tussenstap (voorlopig) nodig)
+    coll:IMBOR restapi:hasMember ?uriConcept .
+    coll:IMBOR-versie-2022 restapi:hasMember ?uriConcept .
+    imbor: restapi:hasMember ?uriConcept .
+
+# Selecteer Label, Synoniem, Definitie, Toelichting waar aanwezig. 
+    OPTIONAL{?uriConcept skos:prefLabel ?labelConcept}
+    OPTIONAL{?uriConcept skos:definition ?definitieConcept}
+    OPTIONAL{?uriConcept skos:note ?toelichtingConcept}
+
+# Indien mogelijk laat het quantitykind erbij zien (grootheid)
+    OPTIONAL{?uriConcept nen2660:hasQuantityKind ?uriGrootheid }
```

Comment:
QUDT hebben we zeker niet ingeladen in de graaf die dit beSPARQLd? Anders had je hier de leesbare naam van de grootheid neer kunnen zetten.

Reply:
Klopt